packages/core/package.json

```diff
@@ -15,8 +15,14 @@
   },
   "files": [
     "lib",
-    "android",
-    "ios",
+    "android/src",
+    "android/build.gradle",
+    "android/gradle.properties",
+    "ios/RNEmbrace/EmbraceManager.h",
```

Comment:
does anything unwanted get added if we use `ios/RNEmbrace` here instead? I'm just a bit worried about forgetting to update this when add/remove files like we did on the 5.0 branch

Reply:
Yes, the info.plist for now.